### DISCUSSION TITLE
GitHub Education Student Benefits

### DISCUSSION THREAD

**Discussion by calebchongsj** [community/community#146207] [GitHub Education]
### Hi everyone,

I have still yet to receive my GitHub education benefits as a college student. Can someone from the Education Support Team kindly process my application as soon as possible?

**Comment by Zevhys** [comment]
yeah up me too, im still waiting 1 day now

**Comment by cachedcake** [comment]
Hello! @calebchongsj, applications are generally processed within a few days but may take longer in peak periods such as the start of a new semester. 

> For more information resort to the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion, let me know if this helps!

**Comment by AmazingK2k3** [comment]
Hello, Same issue for me as well. I reverified my university credentials on December 1st ( been an github education user for a while) and yet to receive the educational benefits email and I am still signed out of the plans. Can someone from the educational support team follow up on my application as well please?

**Comment by davevad93** [comment]
Hi @calebchongsj ,

GitHub no longer provide manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @calebchongsj was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
